app/views/roles/_permissions.html.erb

```diff
@@ -28,29 +28,29 @@ See COPYRIGHT and LICENSE files for more details.
 ++#%>
 <% permissions.each do |mod, mod_permissions| %>
   <% global_prefix = show_global_role ? "fieldset--global--" : "fieldset--" %>
-  <% module_name = mod.blank? ? "fieldset--global--" + Project.model_name.human.downcase.tr(" ", "_") : global_prefix + l_or_humanize(mod, prefix: "project_module_").downcase.tr(" ", "_") %>
+  <% module_name = mod.blank? ? "fieldset--global--#{Project.model_name.human.downcase.tr(' ', '_')}" : global_prefix + l_or_humanize(mod, prefix: "project_module_").downcase.tr(" ", "_") %>
   <% module_id = module_name.parameterize %>
-  <fieldset class="form--fieldset -collapsible" id="<%= module_id %>">
-    <legend class="form--fieldset-legend">
+
+  <%= render Primer::OpenProject::CollapsibleSection.new(id: module_id, display: :block, mb: 3) do |section| %>
+    <% section.with_title do %>
       <% if show_global_role && mod.blank? %>
         <%= t(:label_global) %>
       <% else %>
         <%= permission_header_for_project_module(mod) %>
       <% end %>
-    </legend>
-    <div class="form--toolbar">
-      <span class="form--toolbar-item">
-        (<%= check_all_links module_id %>)
-      </span>
-    </div>
-    <% unless I18n.t("#{mod}.permission_header_explanation", default: '').empty? %>
-    <div class="-italic">
-      <p>
-        <%= I18n.t("#{mod}.permission_header_explanation").html_safe %>
-      </p>
-    </div>
     <% end %>
-    <div class="-columns-2">
```

Comment:
The two column layout was indeed intended, so I guess we should not simply remove it